build/specifications/NuGet.json

```diff
@@ -358,6 +359,105 @@
         ]
       }
     },
+    {
+      "postfix": "Install",
+      "commonPropertySets": [
+        "sources"
+      ],
+      "definiteArgument": "install",
+      "settingsClass": {
+        "properties": [
+          {
+            "name": "PackageID",
+            "type": "string",
+            "format": "{value}",
+            "help": "Name of the package to install"
+          },
+          {
+            "name": "DependencyVersion",
+            "type": "DependencyVersion",
+            "format": "-DependencyVersion {value}",
+            "help": "<em>(4.4+)</em> The version of the dependency packages to use, which can be one of the following: <ul><li><c>Lowest</c>: (default) the lowest version</li><li><c>HighestPatch</c>: the version with the lowest major, lowest minor, highest patch</li><li><c>HighestMinor</c>: the version with the lowest major, highest minor, highest patch</li><li><c>Highest</c>: the highest version</li><li><c>Ignore</c>: No dependency packages will be used</li></ul>"
```

Comment:
The `<ul>`'s etc didn't work as well as expected:
![image](https://user-images.githubusercontent.com/373389/75614721-b39bbb00-5b8f-11ea-9506-71438bf7b542.png)

Is this what you were expecting, or am I holding it wrong?

Reply:
That’s VSCode I suppose?

Reply:
The one above was Rider. 

This one is VSCode:
![image](https://user-images.githubusercontent.com/373389/75634089-9c77ce80-5c5e-11ea-996e-9d872d517691.png)

This one is VS2019:
![image](https://user-images.githubusercontent.com/373389/75634131-f6789400-5c5e-11ea-8799-62bf96a866c6.png)

I'm not convinced that `<ul>`'s and `<li>`'s are supported..

Reply:
Ahh... it's because you just use the tooltip. In that case, yes that doesn't work. I don't use tooltips, since I don't use the mouse, but if you use `Quick Documentation` in Rider, you'll have a nice output:

<img width="648" alt="image" src="https://user-images.githubusercontent.com/5005566/75640448-6eca5f80-5c35-11ea-8503-fdd2acef8258.png">
<img width="870" alt="image" src="https://user-images.githubusercontent.com/5005566/75640476-843f8980-5c35-11ea-8c27-4fc3011ae3f5.png">

Reply:
And about VisualStudio and VSCode I don't care so much :)